packages/html/ds/src/error-text/error-text.html

```diff
@@ -8,7 +8,14 @@
       xs:gi-text-sm md:gi-text-md
     {% endif %}
   {% endmacro %}
-  <p class="{{ getSizeClass(props.size) | trim }} gi-font-bold gi-leading-5 gi-text-red-600 gi-clear-both gi-block !gi-mb-[14px] !gi-mt-0">
+
+  {% macro noMarginMacro(noMargin) %}
```

Comment:
Instead of having this particular prop, how about overriding or extending the classes?

@pregno @BogdanDanilescu

Reply:
we could do that as well, but that will mean to give the user the ability to completely override our styles. not sure if we want to do that

Reply:
We can have class names as props but not expose it on storybook.

Reply:
Sounds good to me . @pregno ?

Reply:
fixed. I added the option to add additional classes if needed